### PR TITLE
PTI-126-3: public-nrced UI updates, move models to common.

### PR DESCRIPTION
**Pull request by NickPhura**
Lots of files, but just largely due to adding/moving models.

## Models
- Added models for the flavours.
- Moved the few existing models into the projects/common project.
- Updated model import references accordingly.

## Common
 - Removed some stock files that were created when this project was first created.

## Public-nrced - Details UI
 - Updated/Added order-detail and inspection-detail components

## Public-nrced - Other UI
 - Removed the side-nav
   - removed/adjusted some css as a result
 - Added links in the header bar

## Api - search.js
 - add NRCED flavour schemas to array of flavours

## Global - search.service
 - updated the parameter checking to be more strict.

### CODE REVIEW

File: angular/projects/public-nrpti/src/assets/styles/layout/layout.scss [hunk 1]
```diff
@@ -8,10 +8,10 @@ app-root {
   flex-direction: column;
 }
 
-div.content {
-  transition: 0.25s all;
-  height: 1000px;
-}
+// div.content {
```
Comment (marklise) [inline angular/projects/public-nrpti/src/assets/styles/layout/layout.scss]: Just toss it entirely, no need to comment out

File: angular/projects/public-nrpti/src/assets/styles/layout/layout.scss [hunk 2]
```diff
@@ -185,12 +140,14 @@ main {
...
+  display: flex;
+  top: 57px;
   left: 0;
   right: 0;
+  padding: 0.38rem 1rem;
   background: #e9ecef;
-  padding: 10px 10px 12px 38px;
   position: fixed;
-  transition: 0.25s all;
+  // transition: 0.25s all;
```
Comment (marklise) [inline angular/projects/public-nrpti/src/assets/styles/layout/layout.scss]: Just toss it entirely, no need to comment out

File: angular/projects/public-nrpti/src/assets/styles/layout/layout.scss [hunk 3]
```diff
@@ -211,29 +168,25 @@ main {
...
-      margin-bottom: 0;
-    }
+    padding: 0.65rem 1.25rem;
   }
 
-  @media screen and (max-width: 768px) {
-    position: relative;
-    top: 0;
-    padding-left: 45px;
+  // @media screen and (max-width: 1200px) {
```
Comment (marklise) [inline angular/projects/public-nrpti/src/assets/styles/layout/layout.scss]: Just toss it entirely, no need to comment out

Comment (NickPhura) [reply angular/projects/public-nrpti/src/assets/styles/layout/layout.scss]: Roger Roger

**Comment by NickPhura** [comment]
> 
> 
> `npm run test` is throwing an error:
> 
> image

Must have missed that import when I rebased. Will fix.

**Comment by NickPhura** [comment]
Updated to remove the commented out css, and fix the bad model import.

**Pull request merged**